### ISSUE TITLE
Recovery of Wallet instructions unclear

### ISSUE THREAD

**Issue by Not-a-slave-to-fashion** [Adolfcoin/adolfcore-zero#23]
I have saved all the previous details on how to recover a wallet if it is lost (the 25 words used to recover if in case it is lost) However the wallet recovery instructions are not listed.

How did you lose it?
PC crashed, all data lost (despite saving). No logs kept, but this is what displays if attempting to run from extracted .zip file after crash. (OS - Windows 10 Home Premium)

`2017-May-02 19:40:58.909679 Initializing adolfcoin protocol...
2017-May-02 19:40:58.909679 Adolfcoin protocol initialized OK
2017-May-02 19:40:58.909679 Initializing p2p server...
2017-May-02 19:40:59.409790 Set limit-up to 128 kB/s
2017-May-02 19:40:59.409790 Set limit-down to 128 kB/s
2017-May-02 19:40:59.409790 Set limit to 128 kB/s
2017-May-02 19:40:59.409790 ERROR C:/msys64/home/skankhunt42/adolfcore-zero/src/p2p/net_node.inl:133 Exception at [node_server::init_config], what=invalid signature
2017-May-02 19:40:59.409790 ERROR C:/msys64/home/skankhunt42/adolfcore-zero/src/p2p/net_node.inl:373 Failed to init config.
2017-May-02 19:40:59.409790 Deinitializing core...
2017-May-02 19:40:59.409790 Mining has been stopped, 0 finished
2017-May-02 19:40:59.409790 Failed to deinitialize core...
2017-May-02 19:40:59.409790 Mining has been stopped, 0 finished
2017-May-02 19:40:59.409790 Deinitializing adolfcoin_protocol...
2017-May-02 19:40:59.409790 ERROR C:/msys64/home/skankhunt42/adolfcore-zero/src/daemon/main.cpp:271 Exception in main! Failed to initialize p2p server.
`
This problem has never occured before.

tl;dr - recovery of wallet is unclear, and adolfcore unresponsive.

**Comment by Not-a-slave-to-fashion** [comment]
Update - I was able to recover the wallet, however, adolfnode.exe is still unresponsive.

Wallet cannot connect to daemon.
`default_log: "E:\Other stuff\Miner\adolfcore-zero\wallet.log"
adolfcoin
Fork Name: Mein Original
Build Version: 0.1.0 "Redpill Revenge"
Logging at log level 0 to E:\Other stuff\Miner\adolfcore-zero\wallet.log
Specify wallet file name (e.g., wallet.bin). If the wallet doesn't exist, it will be created.
Wallet file name: minewalletacoin.bin
password: [muh pass is for me not you]
Opened wallet: gcz6qA9mQAxdYcPrsu12sqGHMzSofh2Xkem5wmqDbFpshGiVPURCWwUUrDMaZH4kCh1rideQS2hGejLM8HeabNVm1iYBQNz1p1

Error: wallet failed to connect to daemon (http://localhost:11511). Daemon either is not started or passed wrong port. Please, make sure that daemon is running or restart the wallet with correct daemon address.
**********************************************************************
Use "help" command to see the list of available commands.
**********************************************************************
[wallet gcz6qA]:`

**Comment by thefuhrer** [comment]
Someone posted something very similar on the forum and they hadn't removed the adolfcoin data folder, was that you?

This can be resolved by removing the adolfcoin data folder. This is usually located in your home folder. For example, on my computer: ```C:\Users\skankhunt42\AppData\Roaming\adolfcoin```
On some computers it may be in:   
```C:ProgramData\adolfcoin```

**Comment by ferretinjapan** [comment]
Wasn't me, no.

**Comment by Not-a-slave-to-fashion** [comment]
Yes, that was me. This case can be closed.
